src/_posts/languages/python/2000-01-01-start.md

```diff
@@ -22,17 +22,18 @@ The following versions of Python are available:
 
 | Python Version | scalingo-20     | scalingo-22     |
 | -------------: | --------------: | --------------: |
-| `3.13`         | up to `3.13.0`  | up to `3.13.0`  |
-| `3.12`         | up to `3.12.7`  | up to `3.12.7`  |
-| `3.11`         | up to `3.11.10` | up to `3.11.10` |
-| `3.10`         | up to `3.10.15` | up to `3.10.15` |
+| **`3.13`**     | up to `3.13.0`  | up to `3.13.0`  |
+| **`3.12`**     | up to `3.12.7`  | up to `3.12.7`  |
+| **`3.11`**     | up to `3.11.10` | up to `3.11.10` |
+| **`3.10`**     | up to `3.10.15` | up to `3.10.15` |
 | `3.9`          | up to `3.9.20`  | up to `3.9.20`  |
 | `3.8`          | up to `3.8.20`  | unsupported     |
 
 {% note %}
 Even though we still support them, we strongly advise against using deprecated
-versions of Python, such as Python `3.7.x`. If you are still using one of these
-deprecated versions, please consider migrating to a more recent version.
+versions of Python, such as Python `3.9.x` and below. If you are still using
+one of these deprecated versions, please consider migrating to a more recent
+version.
```

Comment:
suggestion: I understand why the note here, but with the current information in the doc I don't understand how a customer would know what is a "deprecated" version. What do you think about removing the deprecated Python versions from the table above?

Reply:
I hesitated ^^
I've removed them, and I've also removed the warning in https://github.com/Scalingo/documentation/pull/2845/commits/eef5bba7df4bb36779e490cf402e0e509d3e3d84

Reply:
Nice, I like when we are on the same page :D